mrchecker-framework-modules/mrchecker-core-module/pom.xml

```diff
@@ -163,39 +96,32 @@
         <dependency>
             <!-- Test listener. Binds Junit exec to Allure annotation -->
             <groupId>io.qameta.allure</groupId>
-            <artifactId>allure-junit4</artifactId>
-
+            <artifactId>allure-junit5</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
+            <artifactId>allure-java-commons</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
+            <artifactId>allure-junit-platform</artifactId>
         </dependency>
         <dependency>
             <!-- Allure report generator -->
             <groupId>io.qameta.allure</groupId>
             <artifactId>allure-maven</artifactId>
-            <version>${allure-maven-report-generator.version}</version>
         </dependency>
+
         <dependency>
             <groupId>org.aspectj</groupId>
             <artifactId>aspectjweaver</artifactId>
-            <version>${aspectj.version}</version>
         </dependency>
 
```

Comment:
is there any reason not to define the version of module here?

Reply:
@other019 The inheritance of version numbers was introduced for all modules.